content/changelog/2024-02-27-images-update-part2.md

```diff
@@ -0,0 +1,21 @@
+---
+title: February 2024 images update (part 2)
+date: 2024-02-27
+tags:
+  - images
+  - update
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+description: The best is yet to come
+excludeSearch: true
+---
+
+Earlier this month, we published [a first set of updates](/changelog/2024-02-02-images-update/) for Docker, Erlang, Go, Haskell, Ruby and Rust. Some days ago, we finished this work and all images are now up-to-date. This process occurred with no impact for our users. 
+
+As mentioned in a previous post, this enables [the new `healtcheck` feature](/changelog/2024-02-26-healthcheck-for-everyone/) available for all applications. New images include security patches, Linux kernel 6.7.1, OpenSSL 3.2.1 and Node.js 20.11.0 by default. For the latter, you can change it in the updated images via the `CC_NODE_VERSION` [environment variable](/doc/reference/reference-environment-variables/#commons-to-all-applications).
+
+.Net `8.0` version is now available and enabled by default for new applications. This value can be modified via [the `CC_DOTNET_VERSION` environment variable](/doc/reference/reference-environment-variables/#net). PHP 8.3 is next to come, we're currently working on it. The Elastic / APM agent has been updated for Java users and is now compatible with Java 21. 
```

Comment:
Same here, not very readable and everything is mixed up (dotnet, php, apm...).

I prefer something simpler, like :

Dotnet:

- Version 8 support
- CC_DOTNET_VERSION variable now created by default for each new Dotnet application

Java:
- Version bump to 1.47.1 (With Java 21 support)